internal/maps/maps.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package maps
+
+import (
+	"bytes"
+	"encoding/gob"
+)
+
+func CopyMap(m map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
If there is a better solution for deep-copying a map with arbitrary values in go, let me know.